controller/lib/core/blower_fsm.h

```diff
@@ -84,6 +84,10 @@ struct BlowerSystemState {
   bool is_new_breath = false;
 };
 
+// Transition from PEEP to PIP pressure over this length of time.  Citation:
+// https://respiraworks.slack.com/archives/C011CJQV4Q7/p1591763842312500?thread_ts=1591759016.310200&cid=C011CJQV4Q7
+static constexpr Duration RISE_TIME = milliseconds(100);
```

Comment:
`static` global functions or global variables should only appear in cc files.  In header files, you want to mark global functions or global variables as `inline`.

`static` means "private to this translation unit`, so a static variable/function in a header gets duplicated into every cpp that includes it!

(To maximize confusion, `static` inside of a class means something entirely different.  So this variable, previously inside of a class, was correctly marked as `static inline`.  Whereas variables outside of a class should never be `static inline`.)

Reply:
Woops, hasty cut/paste on my part. Thanks for the clarification as to why it was static inline to begin with.